### PR TITLE
Add wikidata footer link #426

### PR DESCRIPTION
**Pull request by EeveesEyes**
Adds small image to footer - implements #426 

Before:
tmp
After:
tmp2

### CODE REVIEW

File: app/src/app/core/components/footer/footer.component.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <footer class="text-center">
   <div class="container">
-    <div class="d-flex flex-wrap justify-content-center">
+    <div class="d-flex flex-wrap justify-content-center align-items-center">
       <a class="p-3" [routerLink]="['/about']" i18n="Menu About item label@@menuAbout">About</a>
       <a class="p-3" [routerLink]="['/imprint']" i18n="Menu Legal Notice item label@@menuLegalNotice">Legal Notice</a>
       <a class="p-3" [routerLink]="['/data-protection']" i18n="Menu Data Protection item label@@menuDataProtection">Data Protection Declaration</a>
+      <a class="p-3 " href="https://wikidata.org" target="_blank">
```
Comment (jschurse) [inline app/src/app/core/components/footer/footer.component.html:7]: Nice you thought of target="_blank"!

File: app/src/app/core/components/footer/footer.component.html [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+        <img class="wikidata-link" src="https://upload.wikimedia.org/wikipedia/commons/a/ae/Wikidata_Stamp_Rec_Dark.svg" alt="Wikidata_Stamp_Rec_Dark.svg">
```
Comment (jschurse) [inline app/src/app/core/components/footer/footer.component.html:8]: Do we want to load this image from an remote server?
If the server takes time to answer our pageload will be slowed

Comment (EeveesEyes) [reply app/src/app/core/components/footer/footer.component.html:8]: Not sure, should we add it to our assets?

**Comment by jschurse** [comment]
We should keep this in mind if we see a decline in our page loading time.

Am Di., 30. Juni 2020 um 12:52 Uhr schrieb Adrian Kailus <
notifications@github.com>:

**Pull request merged**